Symfony/CS/Tests/Fixer/Contrib/MethodArgumentDefaultValueFixerTest.php

```diff
@@ -0,0 +1,136 @@
+            ),
+            array(
+                <<<'EOT'
+                    <?php
+                        function eFunction($foo, $bar, \SplFileInfo $baz, $x = 'default') {};
+
+                        function fFunction($foo, $bar, \SplFileInfo $baz, $x = 'default') {};
+EOT
+                ,
+                <<<'EOT'
+                    <?php
+                        function eFunction($foo, $bar, \SplFileInfo $baz, $x = 'default') {};
+
+                        function fFunction($foo, $bar = 'removedValue', \SplFileInfo $baz, $x = 'default') {};
+EOT
+            ),
+            array(
+                '<?php function foo ($bar /* a */ /* b */, $c) {}',
+                '<?php function foo ($bar /* a */ = /* b */ 1, $c) {}',
+            ),
+            array(
+                '<?php function hFunction($foo,$bar,\SplFileInfo $baz,$x = 5) {};',
+                '<?php function hFunction($foo,$bar=\'removedValue\',\SplFileInfo $baz,$x = 5) {};',
+            ),
+            array(
+                '<?php function eFunction($foo, $bar, \SplFileInfo $baz = null, $x) {}',
+                '<?php function eFunction($foo = PHP_EOL, $bar, \SplFileInfo $baz = null, $x) {}',
+            ),
+            array(
+                '<?php function eFunction($foo, $bar) {}',
+                '<?php function eFunction($foo       = null, $bar) {}',
+            ),
+            array(
+                <<<'EOT'
+                    <?php
+                        function foo(
+                            $a, // test
+                            $b, /* test */
+                            $c, // abc
+                            $d
+                        ) {}
+EOT
+            ,
+                <<<'EOT'
+                    <?php
+                        function foo(
+                            $a = 1, // test
+                            $b = 2, /* test */
+                            $c = null, // abc
+                            $d
+                        ) {}
+EOT
+            ),
+            array(
+                '<?php function foo($foo, $bar) {}',
+                '<?php function foo($foo = array(array(1)), $bar) {}',
+            ),
+            array(
+                '<?php function a($a, $b) {}',
+                '<?php function a($a = array(\'a\' => \'b\', \'c\' => \'d\'), $b) {}',
+            ),
+            array(
+                '<?php public function a($a, $b) {}',
+                '<?php public function a($a = [\'a\' => \'b\', \'c\' => \'d\'], $b) {}',
+            ),
+            array(
+                '<?php public function a($a, $b) {}',
+                '<?php public function a($a = NULL, $b) {}',
+            ),
+            array(
+                '<?php public function a($a, $b) {}',
+                '<?php public function a($a = Null, $b) {}',
+            ),
+            array(
+                '<?php public function a(&$a, $b) {}',
+                '<?php public function a(&$a = null, $b) {}',
```

Comment:
I am pretty sure it was absolutely unnecessary to outline the same thing 5 times.